core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -221,8 +220,22 @@
   GENERAL_MAX_MESSAGE_SIZE("general.server.message.size.max", "1G", PropertyType.BYTES,
       "The maximum size of a message that can be sent to a server."),
   GENERAL_SIMPLETIMER_THREADPOOL_SIZE("general.server.simpletimer.threadpool.size", "1",
-      PropertyType.COUNT, "The number of threads to use for " + "server-internal scheduled tasks"),
-  // If you update the default type, be sure to update the default used for initialization failures
+      PropertyType.COUNT, "The number of threads to use for " + "server-internal scheduled tasks"), // If
+                                                                                                    // you
+                                                                                                    // update
+                                                                                                    // the
+                                                                                                    // default
+                                                                                                    // type,
+                                                                                                    // be
+                                                                                                    // sure
+                                                                                                    // to
+                                                                                                    // update
+                                                                                                    // the
+                                                                                                    // default
+                                                                                                    // used
+                                                                                                    // for
+                                                                                                    // initialization
+                                                                                                    // failures
```

Comment:
I'm not sure that this comment change is an improvement. Was this intended, or a formatting issue?